x11-misc/polybar/polybar-3.5.2.ebuild

```diff
@@ -31,6 +31,10 @@ DEPEND="
 	x11-libs/xcb-util-xrm
 	alsa? ( media-libs/alsa-lib )
 	curl? ( net-misc/curl )
+	doc? (
+		dev-python/sphinx
+		dev-vcs/git
```

Comment:
We actually *need* git?

Reply:
Not anymore! Upstream patch added (only affects build time).